src/App.vue

```diff
@@ -39,29 +39,53 @@
 export default {
   data: () => ({
     lists:[],
-    newTask:''
+    newTask:'',
+    displayState:''
   }),
+  computed:{
+    viewTodos(){
+      return this.displayState
+      ? this.lists.filter((i) => i.status === this.displayState)
+      : this.lists
+    }
+  },
   methods:{
     // Todoの追加
     addList:function(e){
       e.preventDefault();
       if (this.newTask.match(/\S/g)) 
         this.lists.push({
+          id: this.lists.length,
           task: this.newTask,
           status:"作業中",
       });
+      this.newTask=''
```

Comment:
=の前後には、他と同様にスペースを空けましょう。
```suggestion
      this.newTask = ''
```